build/tasks/docs.js

```diff
@@ -6,45 +6,57 @@ const glob = require('glob');
 const del = require('del');
 const vinylPaths = require('vinyl-paths');
 const docjs = require('documentation');
+const REGEX_FILE_NAME = /((\w)+(-)*(\w))+(?=.js)/g;
```

Comment:
I checked it again, as it was my regex (feels over-engineered :D).

Could you just try out this one: ` /((-)*(\w))+(?=.js)/g`instead of the one above. What it basically does is, it 'captures' all words of a filename until ".js". It also checks for a beginning dash of the name so it can capture the whole filename. If we would just use "\w (word)" it would only catch the first ws and stop.

We could write some code but this would be:

- Split filename `split(-)`;
- Split the last Element of this Array again `split(.)` as array would be `['ws', 'input', 'edit.js']`
- create a new Array combining both
- `.join(-)` to get "ws-input-edit".

But I thought a one line regex makes sense here instead of doing this slicing, splitting thing ^^.